code/modules/clothing/spacesuits/rig/suits/station.dm

```diff
@@ -200,6 +200,12 @@
 		bio = ARMOR_BIO_SHIELDED,
 		rad = ARMOR_RAD_SHIELDED
 	)
+	offline_slowdown = 2
+	offline_vision_restriction = TINT_NONE
+
+	allowed = list(/obj/item/device/flashlight,/obj/item/tank,/obj/item/device/suit_cooling_unit,/obj/item/gun,/obj/item/ammo_magazine,/obj/item/ammo_casing,/obj/item/melee/baton,/obj/item/melee/energy/sword,/obj/item/handcuffs)
+	
+	allowed_module_types = MODULE_GENERAL | MODULE_LIGHT_COMBAT | MODULE_HEAVY_COMBAT | MODULE_UTILITY
```

Comment:
The light combat and heavy combat module types will have to be removed on this.

Reply:
This is more or less the entire point of this PR. Allowing it to have weapons installed makes expeditions less of a pain in the ass to host as you don't have to steal half the armory or wait half the round for cargo to get generally subpar guns in either.

Reply:
The hardsuit modules would have to be produced by robotics either way and that point it's more appropriate to actually give the BC's a hazard or combat hardsuit if absolutely required. There is no reason to let this hardsuit do everything and that was not the reason I asked Danse to add it.

Reply:
As I already said, the reason is to let it get an energy gun if the armory or cargo isn't an option. The biggest pain in the ass involved with expeditions is getting the stuff to actually do them, and the longer that part takes, the less of a chance the expedition happens and actually gets back to the ship before round end. It only has melee armor values that are wroth anything, so its only good in an actual fight against simple mobs or people without guns. I don't really see a way you could powergame with this suit that couldn't also be done with an actual combat hardsuit, since iirc the price to make this and the hazard are similar in the machinist's lab and you still NEED a machinist to actually put a gun on it.